core/renderers/common/constants.js

```diff
@@ -1166,7 +1166,8 @@ Blockly.blockRendering.ConstantProvider.prototype.getCSS_ = function(selector) {
   return [
     /* eslint-disable indent */
     // Fields.
-    selector + ' .blocklyText {',
+    selector + ' .blocklyText, ',
+    selector + ' .blocklyFlyoutLabelText {',
```

Comment:
I would prefer that this additional selector `selector + ' .blocklyFlyoutLabelText` not be added here and instead
add font-family, font-size, font-weight to the block with the flyout labels selector you added on L1187. 

This would not change the css that is applied, but it would make the css rules cleaner. Currently the flyout label has fill colors #fff and #000 in the rules (with the second declared color being what is actually applied due to ordering). So it's harder to scan to see which the css applied. Also I think it would match the comments better.

Same comment applies to the `Zelos.getCSS_`

Reply:
Alternately I could do something like this: 
```
.blocklyText, .blocklyFlyoutLabelText { ... TEXT FONTS ... }

.blocklyText { fill: #000 }
.blocklyFlyoutLabelText { fill: #fff }
```

Reply:
That seems fine.
With the alternative you're suggesting I would also recommend updating the comments in the CSS because at the moment the comments separate sections of CSS rules like `Fields` and I don't think that `blocklyFlyoutLabelText` would fit in `Fields`.